### ISSUE TITLE
test, sdk: socket hangup in sdk integration tests

### ISSUE THREAD

**Issue by yorkie** [alibaba/pipcook#325]
See https://github.com/alibaba/pipcook/runs/853093240, the detailed error is:

```
pipeline api.job test
    ✓ prepare
    ✓ create pipeline
    ✗ create job
      - Error: socket hang up
    ✗ query job info
      - TypeError: Cannot read property 'id' of undefined
    ✗ list job infos
      - TypeError: Cannot read property 'id' of undefined
    ✗ clean
      - TypeError: Cannot read property 'status' of undefined
```

**Comment by FeelyChau** [comment]
fix PR: https://github.com/alibaba/pipcook/pull/326

**Comment by FeelyChau** [comment]
All the apis and sdk interface have been refactored by #408

**Comment by FeelyChau** [comment]
Merged into master #405 .
